.github/workflows/build.yml

```diff
@@ -50,13 +50,13 @@ jobs:
         run: dotnet build --configuration release
 
       - name: Publish Bicep
-        run: dotnet publish --configuration release --self-contained true -p:PublishTrimmed=true -p:PublishSingleFile=true -p:TrimmerDefaultAction=copyused -p:SuppressTrimAnalysisWarnings=true -r ${{ matrix.rid }} ./src/Bicep.Cli/Bicep.Cli.csproj
+        run: dotnet publish --configuration release --self-contained true -r ${{ matrix.rid }} ./src/Bicep.Cli/Bicep.Cli.csproj
```

Comment:
How come we removed `SuppressTrimAnalysisWarnings` and `PublishSingleFile`?

Reply:
Actually never mind, I just saw the changes in the *.csproj file so they were just moved which makes sense :)